.travis.yml

```diff
@@ -11,7 +11,14 @@ env:
     - PIGPIO_ARTIFACT="$PIGPIO_GROUP/$PIGPIO_NAME/$PIGPIO_VERSION/$PIGPIO_NAME-$PIGPIO_VERSION"
   
 script:
-  - docker run -v "$TRAVIS_BUILD_DIR:/data" jwiii/jnaerator:0.12 java -jar jnaerator.jar -mode StandaloneJar -runtime JNA -jar "/data/build/$PIGPIO_ARTIFACT.jar" /data/pigpio/pigpio.h
+  - > 
+      docker run -v "$TRAVIS_BUILD_DIR:/data" jwiii/jnaerator:0.12 
+      java -jar jnaerator.jar 
+      -mode StandaloneJar -runtime JNA 
+      -jar "/data/build/$PIGPIO_ARTIFACT.jar" 
+      -library pigpio
+      -package pigpio.javadsl
```

Comment:
Pattern the package name after Akka Streams here.

The Scala companion library will then be `pigpio.scaladsl`.